.github/workflows/release-pr.yml

```diff
@@ -8,16 +8,23 @@ on:
         type: choice
         options:
           - aligned-sized
-          - light-macros
-          - light-sparse-merkle-tree
+          - light-bounded-vec
+          - light-circuitlib-rs
+          - light-concurrent-merkle-tree
+          - light-hash-set
           - light-hasher
-          - light-merkle-tree-program
-          - light-psp2in2out
-          - light-psp10in2out
-          - light-psp4in4out-app-storage
-          - light-psp2in2out-storage
-          - light-verifier-sdk
-          - macro-circom
+          - light-indexed-merkle-tree
+          - light-macros
+          - light-merkle-tree-event
+          - light-merkle-tree-reference
+          - light-protocol
```

Comment:
psp-protocol

Reply:
@SwenSchaeferjohann made a good point that psp is not a good fit anymore, let's talk about a different prefix in team call or just use light